frontend/src/app/projects/project-detail/project-users/project-user-settings.component.ts

```diff
@@ -59,9 +63,19 @@ export class ProjectUserSettingsComponent implements OnChanges {
   constructor(
     public projectUserService: ProjectUserService,
     public userService: UserService,
-    private toastService: ToastService
+    private toastService: ToastService,
+    private projectService: ProjectService
   ) {}
 
+  ngOnInit(): void {
+    this.projectService.project
+      .pipe(untilDestroyed(this), filter(Boolean))
+      .subscribe((project) => {
+        this.projectSlug = project!.slug;
+        this.projectName = project!.name;
```

Comment:
In this case, I'd just save the project itself. Makes it a lot easier if we want to extend it in the future, otherwise this method will grow for each new attribute.